includes/class-wc-quipu-integration.php

```diff
@@ -391,8 +391,12 @@ public function sync_orders_callback() {
 	 * Create a Quipu invoice, for a given order, prefix and completed date
 	 */
 	public function create_quipu_invoice( $order_id , $prefix, $completed_date) {
+		// error_log ( 'completed date: ' . $completed_date );
+
+		$wk_order = wc_get_order( $order_id );
 
-		$quipu_invoice_id = get_post_meta($order_id, "_quipu_invoice_id", true);
+		$quipu_invoice_id = $wk_order->get_meta( '_quipu_invoice_id', true );
+		// error_log( 'invoice id:' . $quipu_invoice_id );
```

Comment:
I'd remove this commented part (and a few others I see added in this PR), unless we plan to use them very soon

Reply:
They are there within the code in case we need to debug so we kept them, let me know if that is alright.

Reply:
OK! Go ahead then.